src/components/features/TopNavbarTab.vue

```diff
@@ -0,0 +1,22 @@
+<script>
+import { defineComponent } from 'vue';
+export default defineComponent({
+});
+</script>
```

Comment:
Fun fact, if you are not providing any options here - no props, data, methods, anything.
We could skip the entire `<script>...</script>` part 😄

Reply:
Good to know, thanks! 😄